worldengine/model/world.py

```diff
@@ -132,11 +202,9 @@ def _from_protobuf_quantiles(p_quantiles):
 
     @staticmethod
     def _from_protobuf_matrix_with_quantiles(p_matrix):
```

Comment:
I agree with that but I would not do that as part of this PR because this is already big enough :)